app/scripts/controllers/createPersistentVolumeClaim.js

```diff
@@ -111,6 +111,10 @@ angular.module('openshiftConsole')
         if ($scope.claim.storageClass && $scope.claim.storageClass.metadata.name !== "No Storage Class") {
           //we can only have one storage class per claim
           pvc.metadata.annotations["volume.beta.kubernetes.io/storage-class"] = $scope.claim.storageClass.metadata.name;
+          var annotation = $filter('storageClassAccessMode');
```

Comment:
`annotation` is no longer an appropriate var name since we're using a different filter.

```js
var storageClassAccessMode = $filter('storageClassAccessMode');
```